apps/meteor/client/omnichannel/monitors/MonitorsTable.tsx

```diff
@@ -48,7 +48,7 @@ const MonitorsTable = () => {
 
 	// TODO: implement endpoints for monitors add/remove
 	const removeMonitor = useMethod('livechat:removeMonitor');
-	const addMonitor = useMethod('livechat:addMonitor');
+	const addMonitor = useEndpoint('POST', '/v1/livechat/monitors.save');
```

Comment:
You could use `useEndpointMutation` and removed the need for `useMutation` in L80